### ISSUE TITLE
Documentation for snapPoints prop is confusing

### ISSUE THREAD

**Issue by TheWirv** [osdnk/react-native-reanimated-bottom-sheet#234]
Hi, I am a bit confused about the doc for the `snapPoints` prop. At the end of the description there it says, "Note: Array values must be in descending order." Why does it say that? As far as I understand this, I have to provide the snap points like `[450, 300, 125, 0]`. But, judging from my own experience, this is not necessary, and also the "AppleMusic" example especially doesn't do that either.

@osdnk Could you clear that up for me, please?

**Comment by TheWirv** [comment]
I just looked at the code and noticed that sorting the snap points has been there almost from the beginning ([here](https://github.com/osdnk/react-native-reanimated-bottom-sheet/blob/ffcf810991585ce24a5331369fb95b9d6075ecf2/src/index.tsx#L748) to be exact).
So, in other words, the Note in the `snapPoints` description is deprecated, or is there something I'm missing?

**Comment by TheWirv** [comment]
At least that's how I understood it.

There is an internal proper sorting of the snapPoints happening, which also keeps track of the initial indeces.

**Comment by wmonecke** [comment]
Yes, the documentation is deprecated for that. I will submit a PR.
